build.sbt

```diff
@@ -18,13 +18,14 @@ lazy val microservice = (project in file("."))
   .settings(PlayKeys.playDefaultPort := 9581)
   .settings(
     name := appName,
-    scalaVersion := "2.12.16",
+    scalaVersion := "2.13.10",
     targetJvm := "jvm-1.8",
     libraryDependencies ++= AppDependencies(),
     Test / fork := true,
     retrieveManaged := true,
     // Use the silencer plugin to suppress warnings from unused imports in compiled twirl templates
-    scalacOptions += "-P:silencer:pathFilters=views;routes",
+    //    scalacOptions += "-P:silencer:pathFilters=views;routes",
+    scalacOptions := Seq("-Wconf:src=routes/.*:s", "-Wconf:cat=unused-imports&src=html/.*:s"),
```

Comment:
Don't think we should be setting here as it now ignores the ones brought in by sbt-settings [here](https://github.com/hmrc/sbt-settings/blob/main/src/main/scala/uk/gov/hmrc/DefaultBuildSettings.scala) via enabling sbt-plugin. We should be adding to it. i.e.
```
scalacOptions ++= Seq(
      "-Wconf:src=routes/.*:s",
      "-Wconf:cat=unused-imports&src=html/.*:s"
    ),
```
When running `show scalacOptions` on the sbt shell currently it gives
<img width="668" alt="Screenshot 2023-02-02 at 11 47 49" src="https://user-images.githubusercontent.com/63714619/216382676-d2e79818-6e6e-41b6-adfe-e20a55d97890.png">
and you also have warnings prompting to add `-deprecation ` when via sbt-settings
<img width="506" alt="Screenshot 2023-02-02 at 12 28 33" src="https://user-images.githubusercontent.com/63714619/216383084-223f0828-f63a-402f-a08d-0ab0780b8c6e.png">

Once you made the changes, you might need to run `reload` before running `show scalacOptions`. Thanks

Reply:
Doing this will generate a lot of warnings so just as a heads up. A lot of deprecations etc.